main.go

```diff
@@ -61,7 +62,7 @@ func main() {
 		"Watch custom resources in the namespace, ignore other namespaces. If empty, all namespaces will be watched.")
 	flag.Parse()
 
-	ctrl.SetLogger(zap.Logger(true))
+	ctrl.SetLogger(zap.New(zap.UseDevMode(true)))
```

Comment:
Should we revert this?

Reply:
Not releated to this pr. Just stop using deprecated method

Reply:
reverted